### DISCUSSION TITLE
Multi projects/nav menus in single Wiki.js instance

### DISCUSSION THREAD

**Discussion by iWangJiaxiang** [requarks/wiki#3435] [Help / Questions]
**Question**

Hi, developers. Thank you for bringing us such an awesome oss wiki solution!

If my team has many projects that need to be documented in **Wiki.js**, do I need to deploy multi **Wiki.js** instance for each one of project? 

Is there an elegant way to host projects' docs in single **Wiki.js** instance?

The virtual folder struct could handle the hierarchy of projects but I could not setup multi nav menus for each one the project.

Do you have any suggestion?

**Host Info (please complete the following information):**
OS: Docker
Wiki.js version: 2.5
Database engine: postgres 9.7

**Comment by MarsWarrior** [comment]
Seems you want something like Bookstack is using: shelves and books.

If you open a Book, you only see the menu/toc for that Book.

You can have hundreds of Books without having a single menu for all those books...

**Comment by iWangJiaxiang** [comment]
But I want to setup beautiful nav manus for each one of the “book”. The wiki.js only supports single instance level nav menu

发件人: Mars<mailto:notifications@github.com>
发送时间: 2020年9月10日 下午 8:11
收件人: Requarks/wiki<mailto:wiki@noreply.github.com>
抄送: Wang Jiaxiang<mailto:iwangjiaxiang@outlook.com>; Author<mailto:author@noreply.github.com>
主题: Re: [Requarks/wiki] Multi projects/nav menus in single Wiki.js instance (#2414)


Seems you want something like Bookstack is using: shelves and books.

If you open a Book, you only see the menu/toc for that Book.

You can have hundreds of Books without having a single menu for all those books...

―

**Comment by MarsWarrior** [comment]
Some sort of "namespace" feature would be nice in that case, where you can create a menu per namespace.
A namespace can be a team or group (or in Bookstack terms a shelf).

I also thought that collapsible menu's are coming to WikiJs as you can see here: https://requarks.canny.io/wiki/p/dropdown-navigation-items
That would already greatly enhance the overview on the menu.
Each collapsible menu can be considered a Book in Bookstack terms...
